public/pages/ThreatIntel/components/SetupThreatIntelAlertTriggers/SetupThreatIntelAlertTriggers.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NotificationChannelTypeOptions, ThreatIntelAlertTrigger } from '../../../../../types';
+import React, { useCallback, useEffect, useState } from 'react';
+import { ThreatIntelAlertTriggerForm } from '../ThreatIntelAlertTriggerForm/ThreatIntelAlertTriggerForm';
+import {
+  getNotificationChannels,
+  parseNotificationChannelsToOptions,
+} from '../../../CreateDetector/components/ConfigureAlerts/utils/helpers';
+import { NotificationsService } from '../../../../services';
+import { EuiButton, EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { ThreatIntelIocType } from '../../../../../common/constants';
+import { getEmptyThreatIntelAlertTrigger } from '../../utils/helpers';
+
+export interface SetupThreatIntelAlertTriggersProps {
```

Comment:
Let's consistent with the naming, we can call it `ConfigureThreatlntelAlertTrigger`,this  applies to entire